code/_onclick/click.dm

```diff
@@ -119,9 +119,9 @@
 		return
 
 	if(throw_mode)
-		changeNext_move(CLICK_CD_THROW)
-		throw_item(A)
-		return
+		if(throw_item(A))
+			changeNext_move(CLICK_CD_THROW)
+			return
```

Comment:
```suggestion
		return
```
It should always early return regardless of return value of `throw_item()`

Reply:
Oops my b